sass/components/_categorical-display.scss

```diff
@@ -1,310 +1,359 @@
+            @include transition(right,opacity 0.2s ease-in-out);
 
-            }
-            > input:focus + .ic-icon svg path {
-                fill: $active-color;
-            }
         }
-        .category {
-            padding-top: 2em;
-            &:last-child {
-                padding-bottom: 2.5em;
+        > input:focus + .ic-icon svg path {
+            @include themify($themes) {
+                fill: themed('activeColor');
             }
         }
-        .category-title {
-            color: #333333;
+    }
+    .category {
+        padding-top: 2em;
+        &:last-child {
+            padding-bottom: 2.5em;
         }
-        .category-content {
-            background-color: $DEFAULT-main;
-            margin: 0.4em 1em 0em 1em;
-            border: 1px solid $DEFAULT-border;
-            @include border-radius();
+    }
+    .category-content {
+        margin: 0.4em 1em 0em 1em;
+        @include themify($themes) {
+            background-color: themed('defaultMain');
+            border: 1px solid themed('defaultBorder');
         }
-        .subcategory {
-            font-weight: 500;
-            padding: 0.5em 0 0.5em 1.2em;
-            font-size: 1.1em;
-            color: $DEFAULT-text-strong;
+        @include border-radius();
+    }
+    .subcategory {
+        font-weight: 500;
+        padding: 0.5em 0 0.5em 1.2em;
+        font-size: 1.1em;
+        @include themify($themes) {
+            color: themed('defaultTextStrong');
         }
-        .category-title {
-            display: block;
-            font-size: 1.4em;
-            font-weight: 500;
-            padding: 0.1em 0.7em;
-        }
-        .cq-filter {
-            padding-left: 1.6em;
-            line-height: 3.4em;
-            text-align: left;
-            cursor: pointer;
-            @include transition(background-color 0.3s)
-            @include box-sizing(border-box);
-            &.cq-active-filter {
-                font-weight: 800;
-                color: $active-color;
-                border-right: 2px solid $active-color;
-                background-color: #f2f3f8;
-                .ic-icon {
-                    &.ic-active,
-                    &.ic-forex,
-                    &.ic-commodities,
-                    &.ic-volidx,
-                    &.ic-favorite {
-                        svg > g > circle { fill: $active-color; }
+    }
+    .category-title {
+        display: block;
+        font-size: 1.4em;
+        font-weight: 500;
+        padding: 0.1em 0.7em;
+    }
+    .cq-filter {
+        padding-left: 1.6em;
+        line-height: 3.4em;
+        text-align: left;
+        cursor: pointer;
+        @include transition(background-color 0.3s)
+        @include box-sizing(border-box);
+        &.cq-active-filter {
+            font-weight: 800;
+            @include themify($themes) {
+                color: themed('defaultTextActive');
+                border-right: 2px solid themed('activeColor');
```

Comment:
@shayanbinary Could you please check if `themed(...)` can be used in this way.
I may be wrong but i think `property: value;` syntax dictates `themed(...)` to be the only value.

If that is the case, the fix is obvious
```
border-right-width: 2px;
border-right-style: solid;
@include themify($themes) {
     border-right-color: themed('activeColor');
}
```

Again, it's just an skepticism, let me know if the above works the way it is!

Reply:
@aminroosta  this piece of code fell me in love with `sass-loader` plugin. to be sure I make  sample on `http://shayanbinary.binary.sx/theme-test/` please click on `chart setting` and change theme. I use bellow code
` @include themify($themes) {
     border: 2px solid themed('activeColor');
}`